omui/src/components/code/code.tsx

```diff
@@ -126,7 +126,7 @@ export const CodeEditor = ({
 }: CodeEditorProps) => {
   const [code, setCode] = useState(children);
 
-  useEffect(() => onChange && onChange(code), [code]);
+  useEffect(() => onChange && onChange(code), [code, onChange]);
```

Comment:
I thought you said you wanted to use `onCallback` here. Which do you want?

Reply:
We won't put `useCallback` here, but in defining the function